community/examples/intel/README.md

```diff
@@ -336,7 +335,7 @@ For the purpose of this demo create the container without specifying ACLs. The c
 daos cont create pool1 \
   --label cont1 \
   --type POSIX \
-  --properties rf:1
+  --properties rf:0
```

Comment:
Is this intentional? I don't see a mention in commit and I lack the relevant expertise.